infra/modules/mig_template/main.tf

```diff
@@ -156,6 +156,7 @@ resource "google_compute_instance_template" "tpl" {
 
   lifecycle {
     create_before_destroy = "true"
+    ignore_changes = [ disk[0].source_image, labels ]
```

Comment:
What does this line mean?

Reply:
I think this likely ignores whether image changes or not, whenever you push, the nodes will be updated.

Reply:
Correct, this was to stop any unnecessary redeployments when adding nodes.